metadatastore/commands.py

```diff
@@ -126,8 +126,8 @@ def insert_run_start(time, scan_id, beamline_id, beamline_config, uid=None,
     beamline_id : str
         Beamline String identifier. Not unique, just an indicator of
         beamline code for multiple beamline systems
-    beamline_config : metadatastore.odm_temples.BeamlineConfig
-        Foreign key to beamline config corresponding to a given run
+    beamline_config : str
+        uid of beamline config corresponding to a given run
```

Comment:
I agree with Eric. We do need to get rid off this ambiguity. At this point, we all know the schema very well so it is fine for us but for another person using this library for the first time, it is quite confusing. We could simply upgrade to the new mongoengine library that I belive fixes this issue.